src/sentry/static/sentry/app/components/sidebar/onboardingStatus.tsx

```diff
@@ -55,96 +58,111 @@ class OnboardingStatus extends React.Component<Props> {
       return null;
     }
 
-    const doneTasks = (org.onboardingTasks || []).filter(
-      task => task.status === 'complete' || task.status === 'skipped'
-    );
+    const tasks = getMergedTasks(org);
 
-    const tasks = getOnboardingTasks(org);
     const allDisplayedTasks = tasks.filter(task => task.display);
+    const doneTasks = allDisplayedTasks.filter(isDone);
+    const remaining = allDisplayedTasks.length - doneTasks.length;
+
+    const pendingCompletionSeen = doneTasks.some(
+      task =>
+        allDisplayedTasks.some(displayedTask => displayedTask.task === task.task) &&
+        task.status === 'complete' &&
+        !task.completionSeen
+    );
+
+    const isActive = showPanel && currentPanel === 'todos';
 
-    if (doneTasks.length >= allDisplayedTasks.length) {
+    if (doneTasks.length >= allDisplayedTasks.length && !isActive) {
       return null;
     }
 
-    const tooltipTitle = tct(
-      'Getting started with Sentry: [br] [done] / [all] tasks completed',
-      {
-        br: <br />,
-        done: doneTasks.length,
-        all: allDisplayedTasks.length,
-      }
-    );
-
     return (
       <React.Fragment>
-        <Tooltip title={tooltipTitle} containerDisplayMode="block">
-          <OnboardingProgressBar
-            onClick={onShowPanel}
-            isActive={currentPanel === 'todos'}
-            isCollapsed={collapsed}
-            progress={Math.round((doneTasks.length / allDisplayedTasks.length) * 100)}
+        <Container onClick={onShowPanel} isActive={isActive}>
+          <ProgressRing
+            animateText
+            textCss={progressTextCss}
+            text={allDisplayedTasks.length - doneTasks.length}
+            value={(doneTasks.length / allDisplayedTasks.length) * 100}
+            backgroundColor="rgba(255, 255, 255, 0.15)"
+            size={38}
+            barWidth={6}
           />
-        </Tooltip>
-        {showPanel && currentPanel === 'todos' && (
-          <SidebarPanel
-            collapsed={collapsed}
-            title="Getting Started with Sentry"
-            hidePanel={hidePanel}
-          >
-            <TodoList />
-          </SidebarPanel>
-        )}
+          {!collapsed && (
+            <div>
+              <Heading>{t('Setup Sentry')}</Heading>
+              <Remaining>
+                {tct('[remaining] Remaining tasks', {remaining})}
```

Comment:
This would give the translator the context that the string is preceded by a number
```suggestion
                {tct('[numberRemaining] Remaining tasks', {remaining})}
```

Reply:
Good call. Fixed.